butterfly-cli/src/main/java/com/paypal/butterfly/cli/ButterflyCliOption.java

```diff
@@ -77,6 +78,9 @@
                 .withRequiredArg()
                 .ofType(File.class)
                 .describedAs("result file");
+
+        // In place option
+        optionParser.accepts(CLI_OPTION_IN_PLACE_TRANSFORM, "Transforms the application in the same folder as the original content, cannot be used with output folder option.");
```

Comment:
Actually, if `-f` is used, then `-o` and `-z` are simply ignored (as specified in #14 ). Please change the message to: `"Transforms the application in the same folder as the original content. Options (-o) or (-z) are ignored if (-f) is specified"`